sympy/core/function.py

```diff
@@ -162,6 +161,8 @@ def __init__(cls, *args, **kwargs):
             nargs = (as_int(nargs),)
         cls._nargs = nargs
 
+        super(FunctionClass, cls).__init__(args, kwargs)
```

Comment:
It looks like ManagedProperties and BasicMeta will read this as two args (and empty keyword dict) but it does not matter as they do not use them.

Reply:
Oh I didn't notice that. Obviously it should be `*args, **kwargs`.